---a/net/minecraft/block/state/IBlockState.java
+++b/net/minecraft/block/state/IBlockState.java
@@ -1,11 +1,10 @@
 package net.minecraft.block.state;
 
-import com.google.common.collect.ImmutableMap;
 import java.util.Collection;
+
 import net.minecraft.block.Block;
 import net.minecraft.block.properties.IProperty;
 
-public interface IBlockState
-{
+import com.google.common.collect.ImmutableMap;
+
+public interface IBlockState {
+
     Collection getPropertyNames();
 
     Comparable getValue(IProperty property);
@@ -18,4 +18,4 @@
     ImmutableMap getProperties();
 
     Block getBlock();
-}
+}
